test/scripts/e2e_subs/app-inner-calls-csp.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from goal import Goal
+import algosdk.logic as logic
+
+from datetime import datetime
+
+stamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+print(f"{os.path.basename(sys.argv[0])} start {stamp}")
+
+goal = Goal(sys.argv[1], autosend=True)
+
+joe = goal.new_account()
+
+txinfo1, err = goal.pay(goal.account, joe, amt=500_000)
+assert not err, err
+
+txinfo1, err = goal.keyreg(joe, nonpart=True)
+assert not err, err
+joeb = goal.balance(joe)
```

Comment:
@algoidurovic It seems `joeb` is _not_ referenced.  I confirmed locally that the test passes without balance retrieval.

I'm wondering:
* Can the line be removed?
* I'm _guessing_ it's a copy-paste from `app-inner-calls.py`.  I confirmed the same change works for `app-inner-calls.py`.  If not opposed, can you remove the `app-inner-calls.py` invocation to minimize future ambiguities?

```suggestion

```